### ISSUE TITLE
blind-control: rule input of blind position fixed drop down: change to any value?

### ISSUE THREAD

**Issue by ozett** [rdmtc/node-red-contrib-sun-position#83]
i am experimenting with the blind contol.
i set my opening position to 44, which is fine for my blind-position as open.
now i configure rules and the blind-position in rules is a  drop down with fixed  values.
is it possible to get also a free numerical input for every value from 0-100?
would allow me a much finer setting for the opening degree and position of my blinds.
what do you think?

![image](https://user-images.githubusercontent.com/5937811/71346278-c8444d00-2567-11ea-99a9-8010d6927946.png)

**Comment by Hypnos3** [comment]
The percentage is the range between open and closed position in percent. 
There is already a free numeric input. Change therfore the type of the input field (drop down button on the left side of the input field).

**Comment by ozett** [comment]
numeric input at this place? if its true, than very hard to discover...
![image](https://user-images.githubusercontent.com/5937811/71356669-377b6a80-2583-11ea-90e1-01f56887ff01.png)
but anyway, great!!! and thx for hinting and helping me

**Comment by ozett** [comment]
if i use this free numeric input "[0/9 Level entered]" and set it to "50",
it throws an error:

```
Fehler bei der Ermittlung der Rollladenposition:
Cannot read property 'levelBottom' of undefined
```

![image](https://user-images.githubusercontent.com/5937811/71357240-3c411e00-2585-11ea-9214-8d59d9de683d.png)



if it set it back to 50% from the drowpdown, it works fine like before.
cannot see where to fillin something like "levelBottom"?

**Comment by Hypnos3** [comment]
fixed with Version 1.0.0
